memory/src/main/java/org/opensearch/ml/memory/index/OpenSearchConversationalMemoryHandler.java

```diff
@@ -247,19 +250,25 @@ public ActionFuture<List<ConversationMeta>> getConversations(int maxResults) {
     public void deleteConversation(String conversationId, ActionListener<Boolean> listener) {
         StepListener<Boolean> accessListener = new StepListener<>();
         conversationMetaIndex.checkAccess(conversationId, accessListener);
-
+        log.info("DELETING CID " + conversationId);
```

Comment:
CID is hard to understand if someone who not familiar with this feature look into the log. How about change to "conversation"

Reply:
ack